src/main/java/com/denizenscript/depenizen/bukkit/commands/mythicmobs/MythicSpawnCommand.java

```diff
@@ -43,45 +50,22 @@ public MythicSpawnCommand() {
     // -->
 
     @Override
-    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
-        for (Argument arg : scriptEntry) {
-            if (!scriptEntry.hasObject("location")
-                    && arg.matchesArgumentType(LocationTag.class)) {
-                scriptEntry.addObject("location", arg.asType(LocationTag.class));
-            }
-            else if (!scriptEntry.hasObject("level")
-                    && arg.matchesPrefix("level", "l")
-                    && arg.matchesInteger()) {
-                scriptEntry.addObject("level", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("name")) {
-                scriptEntry.addObject("name", arg.asElement());
-            }
-            else {
-                arg.reportUnhandled();
-            }
-        }
-        if (!scriptEntry.hasObject("location") || !scriptEntry.hasObject("name")) {
-            throw new InvalidArgumentsException("Must specify a name and location.");
-        }
-        scriptEntry.defaultObject("level", new ElementTag(1));
+    public void addCustomTabCompletions(TabCompletionsBuilder tab) {
+        tab.add((Set<String>) MythicMobsBridge.getMobManager().getMobNames());
     }
 
-    @Override
-    public void execute(ScriptEntry scriptEntry) {
-        ElementTag name = scriptEntry.getElement("name");
-        LocationTag location = scriptEntry.getObjectTag("location");
-        ElementTag level = scriptEntry.getElement("level");
-        if (scriptEntry.dbCallShouldDebug()) {
-            Debug.report(scriptEntry, getName(), name, location, level);
-        }
+    public static void autoExecute(ScriptEntry scriptEntry,
+                                   @ArgLinear @ArgName("name") ElementTag name,
+                                   @ArgLinear @ArgName("location") LocationTag location,
+                                   @ArgPrefixed @ArgName("level") @ArgDefaultText("1") ElementTag level,
+                                   @ArgPrefixed @ArgName("reason") @ArgDefaultText("COMMAND") SpawnReason reason) {
```

Comment:
`command`, lowercase

Reply:
Also, are you sure `command` is the proper default? For the `spawn` command the proper default was `custom`.

`command` implies a user-driven `/` command in-game, not a scripted execution.